### ISSUE TITLE
Failing build on Arch linux

### ISSUE THREAD

**Issue by paniash** [Battleman/zoomdl#52]
**Prerequisite**

- [X] I have updated to the latest version,

- [X] I have searched the opened and closed issues for a duplicate,

**Describe the bug**

When installing from the AUR, the binary `zoomdl` does not build. The error log tells me that the package does not pass the validation check.

```sh

~: $ paru -S zoomdl
:: Resolving dependencies...
:: Calculating conflicts...
:: Calculating inner conflicts...

Aur (1) zoomdl-2021.03.28-1

:: Proceed to review? [Y/n]:

:: Downloading PKGBUILDs...
 PKGBUILDs up to date
 nothing new to review
fetching devel info...
==> Making package: zoomdl 2021.03.28-1 (Sat 24 Apr 2021 11:54:12 AM IST)
==> Retrieving sources...
  -> Found zoomdl
==> Validating source files with md5sums...
    zoomdl ... FAILED
==> ERROR: One or more files did not pass the validity check!
:: Packages failed to build: zoomdl-2021.03.28-1
```

**To Reproduce**

Installation using [paru](https://github.com/morganamilo/paru).

**Comment by Battleman** [comment]
OK, you're gonna have to ELI5 me there. I know pacman/paru are equivalents of apt for deb-based systems, and AUR is the repo for Arch. But for debian/ubuntu, you need to publish the package, and it needs to be validated first. I haven't submitted it. I see [it appears](https://aur.archlinux.org/packages/zoomdl/), but I am not the one who submitted it. So is there anything I can do? 

From my basic understanding (based on the error message), there is a md5 mismatch between the version published to AUR and another (which?). If you explain what steps I can do to fix it, or submit a PR that does, I'll gladly look into that, but now I'm kinda lost...

**Comment by paniash** [comment]
@Battleman Hi! Sorry for the late response. So basically, the AUR is the arch equivalent of PPAs in debian/ubuntu systems, in the sense that packages in the AUR are not officially packaged. They are packages published by users and it is strongly recommended to new users to not install any random package from the AUR unless one trusts it.

I tried the default installation method using wget and it works. Looks like the maintainer of the package on the AUR has to fix this build issue. Not an issue from your side afaik. Thanks for this wonderful project btw!

**Comment by Battleman** [comment]
Thanks for the heads up. I'll keep this issue open for now. I'll look if I can ping/takeover the AUR package, or at least document it in the README.

**Comment by rvasilev** [comment]
https://aur.archlinux.org/packages/zoomdl maintainer/packager here. It's building, this one could be closed.
